src/tests/conformance/mod.rs

```diff
@@ -1,29 +1,26 @@
-use std::net::{IpAddr, Ipv4Addr};
+use tempfile::TempDir;
 
 use crate::{
     protocol::codecs::binary::BinaryMessage,
-    setup::{config::ZIGGURAT_DIR, node::NodeBuilder},
+    setup::node::NodeBuilder,
     tools::{config::TestConfig, synth_node::SyntheticNode},
 };
 
 mod handshake;
 mod query;
+mod stateful;
 
 async fn perform_response_test(
     config: TestConfig,
     response_check: &dyn Fn(&BinaryMessage) -> bool,
 ) {
     // Build and start Ripple node
-    let mut node = NodeBuilder::new(
-        home::home_dir()
-            .expect("Can't find home directory")
-            .join(ZIGGURAT_DIR),
-        IpAddr::V4(Ipv4Addr::LOCALHOST),
-    )
-    .unwrap()
-    .build()
-    .await
-    .unwrap();
+    let temp = TempDir::new().expect("Unable to create TempDir");
+    let mut node = NodeBuilder::new(None, temp.path().to_path_buf())
```

Comment:
The path could be the same for each case by default, we're always running the cases in sequence.

Reply:
I guess this comment touches the same issue as above, so my answer still applies here:
`We don't need direct access, but TempDir has special drop impl: it removes all content.
Edit: so IOW it can be done, sure: we would need to do manual cleaning at the end of the test, preferably also at the beginning. On the other hand there would be one dependency less.`

Reply:
As agreed via slack, we're leaving it as is for now.